docs-src/content/_index.md

```diff
@@ -2,11 +2,239 @@
 title: "Cubic SDK Documentation"
 ---
 
-# Cubic SDK
+# Cubic API Overview
 
-Cubic is Cobalt's speech recognition engine.  It can be bundled into your
-application as a library, or deployed on-prem and accessed over the network.
-This documentation refers to accessing the network-based Cubic server.
+Cubic is Cobalt’s automatic speech recognition (ASR) engine. It can be deployed on-prem and accessed over the network or on your local machine via an API. We currently support Go and Python, and are adding support for more languages.
+
+Once running, Cubic’s API provides a method to which you can stream audio. This audio can either be from a microphone or a file. We recommend uncompressed WAV as the encoding, but support other formats such as MP3.
+
+![Cubic setup](img/cubicoverview.png)
+
+Cubic’s API provides a number of options for returning the speech recognition results. The results are passed back using Google’s protobuf library, allowing them to be handled natively by your application. Cubic can estimate its confidence in the transcription result at the word or utterance level, along with timestamps of the words. Confidence scores are in the range 0-1. Cubic’s output options are described below.
+
+## Automatic Transcription Results
+
+The simplest result that Cubic returns is its best guess at the transcription of your audio. Cubic recognizes the audio you are streaming, listens for the end of each utterance, and returns the speech recognition result. 
+
+Cubic maintains its transcriptions in an N-best list, i.e. is the top N transcriptions from the recogniser. The best ASR result is the first entry in this list.
+
+<details>
+<summary>Click here to see an example json representation of Cubic’s N-best list with utterance-level confidence scores</summary>
```

Comment:
The `Click here ... ` line kind of gets lost in the text when viewing the page. Adding a color and bold tag makes it stand out a little : 
```html
<summary><font color="236ecc"><b>Click here to see an example json representation of Cubic’s N-best list with utterance-level confidence scores</b></font></summary>
```
![image](https://user-images.githubusercontent.com/26173976/56456791-301c1580-6393-11e9-8972-9cbdfc3a35ac.png)

Reply:
Thanks